docs/001_develop/03_client-capabilities/013_toast-notifications/index.mdx

```diff
@@ -2,17 +2,450 @@
+      type: 'error', 
+    },
+  },
+  'rapid',
+);
+```
+ `Snackbar` are persistent until dismissed. They can disappear automatically after a short time by setting the `autoClose` property in the snackbar config. 
+
+```typescript
+  showNotificationSnackbar(
+    {
+      body: 'Lorem ipsum',
+      snackbar: {
+        confirmingActions: [{ label: 'Confirm', action: () => console.log('Lorem ipsum') }],
+        type: 'error', 
+        autoClose: true
+      },
+    },
+    'rapid',
+  );
+```
+
+### Dialog
+
+This notification component uses a modal-like window to display notification. `Dialog` notifications requires user to take an action before proceeding as they block the underlying interface until dismissed. Therefore, these are used for critical interactions such as warnings, confirmations or important announcements.  `Dialog` notification component also supports multiple action buttons.
+
+One of the most common use case for dialog is `confirming before deleting`.
+
+An example for using `dialog` component is shown in the Example section above. 
+Multiple action buttons can be shown in a dialog by using `confirmingActions` property as shown in below snippet:
+
+```typescript
+showNotificationDialog(
+  {
+    title: 'Important message',
+    body: 'Lorem ipsum',
+    dialog: {
+      confirmingActions: [
+        { label: 'Confirm', action: () => console.log('Lorem ipsum') },
+        { label: 'Save', action: () => console.log('Lorem ipsum') }
+      ],
+      dismissingAction: {
+        label: 'Dismiss',
+        action: () => console.log('Lorem ipsum'),
+      },
+    },
+  },
+  'rapid',
+);
+```
+
+
+## Notification listener Component
+
+`rapid-notification-listener` component is a key utility in Genesis that listens for notifications event dispatched form any part of your application. It processes the event payload and dynamically displays the appropriate notification component (e.g., `Alert`, `Toast`, `Dialog`, etc.).
+
+By centralizing notification management, the `notification-listener` makes it easy to handle different types of notification in a consistent and modular way.
+
+The client app need to create `notification-listener` at top of the component tree. Any child under `notification-listener` component can dispatch custom **NotificationEvent** event
+
+### Example: Usage of Notification listener
+Lets look at code example to setup and use the `notification-listener` component. You will need to follow the following steps to achieve this:
+
+**1. Wrap your application with the notification-listener**
+
+As discussed earlier `notification-listener` component needs to be placed at the top of component tree to ensure it can capture events from any child component. You can find the `rapid-design-system-provider` tag in your application and place the `notification-listener` tag inside it, whether yu're using React, Angular, or Genesis framework.
```

Comment:
```suggestion
As discussed earlier `notification-listener` component needs to be placed at the top of component tree to ensure it can capture events from any child component. You can find the `rapid-design-system-provider` tag in your application and place the `notification-listener` tag inside it, whether you're using React, Angular, or Genesis framework.
```